package rina.messages;
option java_package="rina.encoding.impl.googleprotobuf.apnaminginfo";

message applicationProcessNamingInfo_t {         // carries the naming information of an application process
        required string applicationProcessName = 1;
        optional string applicationProcessInstance = 2;
        optional string applicationEntityName = 3;
        optional string applicationEntityInstance = 4;
}
